RecoHI/HiTracking/python/hiMixedTripletStep_cff.py

```diff
@@ -1,206 +1,159 @@
-hiMixedTripletSeedsB.ClusterCheckPSet.MaxNumberOfPixelClusters = 5000000
-hiMixedTripletSeedsB.ClusterCheckPSet.MaxNumberOfCosmicClusters = 50000000
-
-import RecoTracker.TkSeedGenerator.GlobalCombinedSeeds_cfi
-hiMixedTripletSeeds = RecoTracker.TkSeedGenerator.GlobalCombinedSeeds_cfi.globalCombinedSeeds.clone()
-hiMixedTripletSeeds.seedCollections = cms.VInputTag(
-    cms.InputTag('hiMixedTripletSeedsA'),
-    cms.InputTag('hiMixedTripletSeedsB'),
-    )
+mixedTripletStepSeedLayersB.BPix.skipClusters = cms.InputTag('hiMixedTripletStepClusters')
+mixedTripletStepSeedLayersB.TIB.skipClusters = cms.InputTag('hiMixedTripletStepClusters')
+
+hiMixedTripletStepTrackingRegionsB = _globalTrackingRegionWithVertices.clone(RegionPSet=dict(
+     precise = True,
+     useMultipleScattering = False,
+     beamSpot = "offlineBeamSpot",
+     useFoundVertices = True,
+     useFakeVertices       = False,
+     VertexCollection = "hiSelectedPixelVertex",
+     useFixedError = True,
+     fixedError = 3.75,#15.0
+     ptMin = 0.4,
+     originRadius = 1.0,
+     originRScaling4BigEvts = cms.bool(True),
+     halfLengthScaling4BigEvts = cms.bool(False),
+     ptMinScaling4BigEvts = cms.bool(True),
+     minOriginR = 0,
+     minHalfLength = 0,
+     maxPtMin = 0.7,
+     scalingStartNPix = 20000,
+     scalingEndNPix = 35000     
+))
+
+
+# seeding
+mixedTripletStepHitDoubletsB.clusterCheck = ""
+mixedTripletStepHitDoubletsB.trackingRegions = "hiMixedTripletStepTrackingRegionsB"
 
 # QUALITY CUTS DURING TRACK BUILDING
-import TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff
-hiMixedTripletTrajectoryFilter = TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
-    maxLostHits = 0,
-    minimumNumberOfHits = 6,
-    minPt = 1.0
-    )
-
-# Propagator taking into account momentum uncertainty in multiple scattering calculation.
-import TrackingTools.MaterialEffects.MaterialPropagator_cfi
-hiMixedTripletPropagator = TrackingTools.MaterialEffects.MaterialPropagator_cfi.MaterialPropagator.clone(
-    ComponentName = 'hiMixedTripletPropagator',
-    ptMin = 1.0
-    )
-import TrackingTools.MaterialEffects.OppositeMaterialPropagator_cfi
-hiMixedTripletPropagatorOpposite = TrackingTools.MaterialEffects.OppositeMaterialPropagator_cfi.OppositeMaterialPropagator.clone(
-    ComponentName = 'hiMixedTripletPropagatorOpposite',
-    ptMin = 1.0
-    )
-
-import TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi
-hiMixedTripletChi2Est = TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi.Chi2MeasurementEstimator.clone(
-    ComponentName = cms.string('hiMixedTripletChi2Est'),
-    nSigma = cms.double(3.0),
-    MaxChi2 = cms.double(16.0)
-    )
-
+from RecoTracker.IterativeTracking.MixedTripletStep_cff import _mixedTripletStepTrajectoryFilterBase
+_mixedTripletStepTrajectoryFilterBase.minimumNumberOfHits = 5
+_mixedTripletStepTrajectoryFilterBase.minPt = 0.4
```

Comment:
This does not work (changes are useless the way they are applied), please apply modifications to mixedTripletStepTrajectoryFilter instead of this internal ``` _mixedTripletStepTrajectoryFilterBase```